### DISCUSSION TITLE
Gatsby mdx plugin does not work with version 3.13.0

### DISCUSSION THREAD

**Discussion by rahulrao0209** [gatsbyjs/gatsby#33656] [Help]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

warn Plugin gatsby-plugin-mdx is not compatible with your gatsby version 3.13.0 - It requires gatsby@^4.0.0-next
warn Plugin gatsby-plugin-mdx is not compatible with your gatsby version 3.13.0 - It requires gatsby@^4.0.0-next

 ERROR 

Error in "C:\Users\rahul\OneDrive\Desktop\UI-Design\projects\bookshelves\bookshelves\node_modules\gatsby-plugin-mdx\gatsby-node.js": Cannot find module 'gatsby/webpack'

### Reproduction Link

---

### Steps to Reproduce

1. Install gatsby plugin mdx - **npm install gatsby-plugin-mdx @mdx-js/mdx @mdx-js/react**
2. Configure the plugin in gatsby-config.js file
3. gatsby develop

### Expected Result

I expect the app to start up as usual.

### Actual Result

Instead I get the following error

warn Plugin gatsby-plugin-mdx is not compatible with your gatsby version 3.13.0 - It requires gatsby@^4.0.0-next
warn Plugin gatsby-plugin-mdx is not compatible with your gatsby version 3.13.0 - It requires gatsby@^4.0.0-next

 ERROR 

Error in "C:\Users\rahul\OneDrive\Desktop\UI-Design\projects\bookshelves\bookshelves\node_modules\gatsby-plugin-mdx\gatsby-node.js": Cannot find module 'gatsby/webpack'

### Environment

```shell
System:
    OS: Windows 10 10.0.19042
    CPU: (8) x64 Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz   
  Binaries:
    Node: 14.17.0 - C:\Program Files\nodejs\node.EXE        
    npm: 6.14.13 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.19041.1266.0), Chromium (95.0.1020.30)
  npmPackages:
    gatsby: ^3.13.0 => 3.13.0
    gatsby-image: ^3.11.0 => 3.11.0
    gatsby-plugin-firebase: ^0.2.0-beta.4 => 0.2.0-beta.4
    gatsby-plugin-image: ^1.13.0 => 1.13.0
    gatsby-plugin-mdx: ^3.0.0 => 3.0.0
    gatsby-plugin-sharp: ^3.13.0 => 3.13.0
    gatsby-plugin-styled-components: ^4.13.0 => 4.13.0
    gatsby-source-filesystem: ^3.14.0 => 3.14.0
    gatsby-transformer-json: ^3.14.0 => 3.14.0
    gatsby-transformer-sharp: ^3.13.0 => 3.13.0
  npmGlobalPackages:
    gatsby-cli: 4.0.0
```

**Comment by LekoArts** [comment]
Hi!

As the warning tells you, you'll need to use `gatsby@4.0.0` with the latest MDX plugin version, or you'll need to downgrade the MDX plugin one major version.

**Reply by rahulrao0209** [reply]
Hi!

Thanks A Lot! It works fine with mdx plugin v2.14.0. I was confused as I believed that v3.0.0 should work with gatsby v3. 
Sorry for reporting that as a bug. 
thanks!
